src/Microsoft.Health.Fhir.R5.Client/Microsoft.Health.Fhir.R5.Client.csproj

```diff
@@ -1,19 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
```

Comment:
Is there a particular reason for removing the blank lines?

Reply:
Just that I'm still using VS2019 and it did it automatically :)